src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -796,24 +796,30 @@ changes_ekey_opts(_StartSeq, #mrargs{end_key=EKey,
     end.
 
 
-reduced_external_size(Tree) ->
+reduced_external_size(btree, Tree) ->
```

Comment:
Matching on the arg is a bit odd here. I'd make this two different functions that you call directly in calculate_eternal_size/1.